### ISSUE TITLE
Fatal Error because namespace ProcessWire

### ISSUE THREAD

**Issue by phlppschrr** [Toutouwai/HannaCodeDialog#4]
I get a `Fatal Error: Uncaught Error: Class 'ProcessWire\TextformatterHannaCode'` when editing pages. After removing `namespace ProcessWire` from `HannaCodeDialog.module` everything works again.

**Comment by Toutouwai** [comment]
Are you running the minimum required PW >= 3.0.0?
If so maybe the file compiler cache needs to be cleared: Modules > Site tab > Clear compiled files.

**Comment by phlppschrr** [comment]
Yes, I am running the latest dev version and already tried clearing the file compiler cache.

**Comment by phlppschrr** [comment]
Instead of removing the ProcessWire namespace I can also change these 2 lines and add a global namespaced `\TextformatterHannaCode` to fix it for me.
```php
$settings['open_tag'] = isset($data['openTag']) ? $data['openTag'] : \TextformatterHannaCode::DEFAULT_OPEN_TAG;
$settings['close_tag'] = isset($data['closeTag']) ? $data['closeTag'] : \TextformatterHannaCode::DEFAULT_CLOSE_TAG;
```

**Comment by Toutouwai** [comment]
Good spotting, thanks. Fixed in v0.2.3.
